lua/nvim-lsp-installer/server.lua

```diff
@@ -70,6 +70,17 @@ function M.Server:setup(opts)
     end
 end
 
+---Attaches this server to all current open buffers with a 'filetype' that matches the server's configured filetypes.
+function M.Server:attach_buffers()
+    log.debug("Attaching server to buffers", self.name)
+    local lspserver = require("lspconfig")[self.name]
+    for _, bufnr in ipairs(vim.api.nvim_list_bufs()) do
+        log.fmt_trace("Attaching server=%s to bufnr=%s", self.name, bufnr)
+        lspserver.manager.try_add_wrapper(bufnr)
+    end
+    log.debug("lspconfig.manager.try_add_wrapper() successfully called", self.name)
+end
+
```

Comment:
@mjlbach :wave: would you say this is correct usage of the server manager? The reason this is needed is because a server may become available after neovim initialization (for example installing a new server), and so we want to attach any open buffers to it

Reply:
I think its fine